libsrc/atari/getdefdev.s

```diff
@@ -68,16 +69,23 @@ crvec:  jsr     $FFFF           ; will be set to crunch vector
         sta     __defdev
         iny
         lda     (DOSVEC),y
-        sta     __defdev+1
+done:   sta     __defdev+1
 
 ; Return pointer to default device
 
 finish: lda     #<__defdev
         ldx     #>__defdev
         rts
 
+; XDOS version
+
+xdos:   lda     XDEFDEV
+        bne     done
+
         .data
 
+crvec:	.byte	$4C,$FF,$FF     ; will be set to crunch vector
```

Comment:
> Yes. So one is as good as the other, isn't it?

No. If it were, we wouldn't need mnemonics and write all our code using .byte ;-))

> If you want, I can change it to "jmp".

Please do so - thanks in advance.